### PR TITLE
[DOCS] GREAT-195 Docs remove all stubs and links to them

### PR DESCRIPTION
**Pull request by petermoyer**
Changes proposed in this pull request:
- GREAT-195

Find all stubs relevant to be removed
find . -type f -exec grep -I stub {} \; -print
For each file, grep for a link reference and remove it
Comment out in the sidebars.js file
Remove from the sitemap, xml

### Definition of Done
Please delete options that are not relevant.

- [x ] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [x ] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [ x] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

**Comment by petermoyer** [comment]
Before reviewing, see a couple of conflicts to resolve

**Comment by petermoyer** [comment]
Conflicts appear merged.. Please take a look

File: sidebars.js [hunk 1]
```diff
@@ -343,14 +336,14 @@ module.exports = {
...
         },
-        //{
-          //type: 'category',
-          //label: 'Spare Parts',
-          //collapsed: true,
-          //items: [
-            //{ type: 'doc', id: 'reference/spare_parts' }
-          //]
-        //},
+        // {
```
Comment (abegong) [inline sidebars.js]: Let's drop this entirely. There's no reason to keep the ancient Spare Parts section.

**Comment by abegong** [comment]
This looks good to me. I made one small suggestion.

Note: I haven't reviewed the XML changes closely.

File: sidebars.js (hunk 1 above)
Comment (petermoyer) [reply sidebars.js]: Ok.. removing that commented section

File: docs/reference/core_concepts.md [hunk 2]
```diff
@@ -54,7 +54,6 @@ of the main concepts and classes you will encounter while using Great Expectatio
 * [Expectation Suite operations](./expectation_suite_operations)
 * [Metrics](./metrics)
 * [Profilers](./profilers)
-* [Stores](./stores)
```
Comment (Shinnnyshinshin) [inline docs/reference/core_concepts.md:57]: Note: This should be `MetaData Stores` when those are linked

File: docs/tutorials/getting_started/customize_your_deployment.md [hunk 3]
```diff
@@ -69,16 +69,13 @@ Great Expectations allows you to connect to data in a wide variety of sources, a
 
 * [How to connect to a Athena database](/docs/guides/connecting_to_your_data/database/athena)
 * [How to connect to a BigQuery database](/docs/guides/connecting_to_your_data/database/bigquery)
-* [How to connect to a MSSQL database](/docs/guides/connecting_to_your_data/database/mssql)
```
Comment (Shinnnyshinshin) [inline docs/tutorials/getting_started/customize_your_deployment.md]: This is currently an open PR.. shouldn't be removed 
https://github.com/great-expectations/great_expectations/pull/3353

File: docs/tutorials/getting_started/customize_your_deployment.md [hunk 4]
```diff
@@ -69,16 +69,13 @@ Great Expectations allows you to connect to data in a wide variety of sources, a
...
 * [How to connect to a MySQL database](/docs/guides/connecting_to_your_data/database/mysql)
 * [How to connect to a Postgres database](/docs/guides/connecting_to_your_data/database/postgres)
 * [How to connect to a Redshift database](/docs/guides/connecting_to_your_data/database/redshift)
 * [How to connect to a Snowflake database](/docs/guides/connecting_to_your_data/database/snowflake)
 * [How to connect to a SQLite database](/docs/guides/connecting_to_your_data/database/sqlite)
 * [How to connect to data on a filesystem using Spark](/docs/guides/connecting_to_your_data/filesystem/spark)
 * [How to connect to data on S3 using Spark](/docs/guides/connecting_to_your_data/cloud/s3/spark)
-* [How to connect to data on GCS using Spark](/docs/guides/connecting_to_your_data/cloud/gcs/spark)
```
Comment (Shinnnyshinshin) [inline docs/tutorials/getting_started/customize_your_deployment.md:80]: This is also an open PR #3365

File: docs/tutorials/getting_started/customize_your_deployment.md [hunk 5]
```diff
@@ -91,8 +88,6 @@ By default, Data Docs are stored locally, in an uncommitted directory. This is g
 ## Additional Checkpoints and Actions
 Most teams will want to configure various Checkpoints and Validation Actions as part of their deployment. There are two primary patterns for deploying Checkpoints. Sometimes Checkpoints are executed during data processing (e.g. as a task within Airflow). From this vantage point, they can control program flow. Sometimes Checkpoints are executed against materialized data. Great Expectations supports both patterns. There are also some rare instances where you may want to validate data without using a Checkpoint.
 
-* [How to update Data Docs as a Validation Action](/docs/guides/validation/validation_actions/how_to_update_data_docs_as_a_validation_action)
```
Comment (Shinnnyshinshin) [inline docs/tutorials/getting_started/customize_your_deployment.md:94]: This exists 
https://docs.greatexpectations.io/docs/guides/validation/validation_actions/how_to_update_data_docs_as_a_validation_action#steps

File: sidebars.js [hunk 6]
```diff
@@ -137,26 +137,26 @@ module.exports = {
                 'guides/connecting_to_your_data/cloud/s3/pandas',
                 'guides/connecting_to_your_data/cloud/s3/spark',
                 'guides/connecting_to_your_data/cloud/gcs/pandas',
-                'guides/connecting_to_your_data/cloud/gcs/spark',
```
Comment (Shinnnyshinshin) [inline sidebars.js:140]: open PR

**Review by Shinnnyshinshin — CHANGES_REQUESTED** [review]
Thank you for this @petermoyer . Requesting some changes

File: docs/reference/core_concepts.md (hunk 2 above)
Comment (petermoyer) [reply docs/reference/core_concepts.md:57]: Ok..  will check in future

File: docs/tutorials/getting_started/customize_your_deployment.md (hunk 3 above)
Comment (petermoyer) [reply docs/tutorials/getting_started/customize_your_deployment.md]: Ok.. Added that back

File: docs/tutorials/getting_started/customize_your_deployment.md (hunk 4 above)
Comment (petermoyer) [reply docs/tutorials/getting_started/customize_your_deployment.md:80]: Added this back

File: docs/tutorials/getting_started/customize_your_deployment.md (hunk 5 above)
Comment (petermoyer) [reply docs/tutorials/getting_started/customize_your_deployment.md:94]: added back

File: sidebars.js (hunk 6 above)
Comment (petermoyer) [reply sidebars.js:140]: Added back

**Review by Shinnnyshinshin — APPROVED** [review]
Thank you @petermoyer. approved once CI/CD is passing

**Pull request merged**